uktrade/data-hub-leeloo
# Add exclude from the investment flow to investment stages metadata

### Description of change
A new field has been added to the investment project stages metadata to indicate if a stage should be excluded from the investment flow. This field will allow a stage to be deprecated without causing disruption to the flow views.

### To Test
- Run migrations `./manage.py migrate`
- Check endpoint shows the new field `http://localhost:8000/metadata/investment-project-stage/`
- Check admin shows the new field `http://localhost:8000/admin/metadata/investmentprojectstage/` 


### Checklist
